package.json

```diff
@@ -27,7 +27,8 @@
     "url": "https://github.com/gribnoysup/react-yandex-maps.git"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^15.5.0",
+    "prop-types": " ^15.5.0"
```

Comment:
`prop-types` [docs](https://github.com/facebook/prop-types#how-to-depend-on-this-package) recommend putting the minimal version at `15.5.7`, could you change that?